tests/test_models.py

```diff
@@ -133,6 +133,15 @@ def test_save_specific_datetime(self):
         k = BarTime.objects.filter(datetime='1398-04-31 12:12:12')
         self.assertEqual(k[0].datetime.day, 31)
 
+    def test_date_lookup_filter(self):
+        jdatetime_ = jdatetime.datetime(1378, 10, 10)
+        bartime = BarTime.objects.create(name="Test", datetime=jdatetime_)
+        self.assertEqual(BarTime.objects.get(datetime__date=jdatetime_), bartime)
+        self.assertEqual(BarTime.objects.get(datetime__date=jdatetime_.date()), bartime)
+        self.assertEqual(BarTime.objects.get(datetime__date="1378-10-10"), bartime)
+        with self.assertRaises(BarTime.DoesNotExist):
+            BarTime.objects.get(datetime__date="1378-10-11")
```

Comment:
I meant a test with an invalid value for `__date`. for example:
```suggestion
        with self.assertRaisesMessage(
            TypeError, 
            "Value of `__date` filter must be an instance of jdatetime.datetime, jdatetime.date or str"
        ):
            BarTime.objects.get(datetime__date=1)

 ```

Reply:
Ahh OK, I misunderstood!